src/dmd/dmodule.d

```diff
@@ -601,10 +601,21 @@ extern (C++) final class Module : Package
     {
         if (FileName.equals(srcfile.toString(), "object.d"))
         {
-            .error(loc, "cannot find source code for runtime library file 'object.d'");
-            errorSupplemental(loc, "dmd might not be correctly installed. Run 'dmd -man' for installation instructions.");
-            const dmdConfFile = global.inifilename.length ? FileName.canonicalName(global.inifilename) : "not found";
-            errorSupplemental(loc, "config file: %.*s", cast(int)dmdConfFile.length, dmdConfFile.ptr);
+            .error(loc, "cannot find source code for runtime library file `object.d`.");
+            errorSupplemental(loc, "DMD might not be correctly installed. Run 'dmd -man' for installation instructions.");
+            if (global.inifilename.length)
+            {
+                const dmdConfFile = FileName.canonicalName(global.inifilename);
+                errorSupplemental(loc, "Perhaps add an include path using `-I` or into the config file (`%.*s`)",
+                        cast(int)dmdConfFile.length,
+                        dmdConfFile.ptr);
+            }
+            else
+            {
+                errorSupplemental(loc, "There is no config file. Perhaps add one or add an include path using `-I`.");
+            }
+            if (global.path)
+                errorSupplemental(loc, "Currently, the compiler found the following include paths:");
```

Comment:
The compiler doesn't "find" include paths, suggests that the wording is misleading.

Reply:
I changed to `search on`. If you know better wording, please tell